colcon_core/package_augmentation/python.py

```diff
@@ -156,3 +162,22 @@ def _next_incompatible_version(version):
     if len(version) == 1:
         version.append(0)
     return '.'.join(map(str, version))
+
+
+def _extract_maintainers_with_emails(metadata):
+    if 'maintainer' in metadata:
+        maintainer = metadata['maintainer']
+        maintainer_email = metadata.get('maintainer_email')
+    else:
+        # If no explicit maintainer is given then it is likely that the
+        # original author is maintaining the package
+        maintainer = metadata.get('author')
```

Comment:
We probably want to check if author exists  before using get.

Reply:
`get` will return `None` if not set, which is intentional, otherwise I'd have used `['author']`.
We check it in 3 lines.